examples/text-editor/ThemePlugin.wyt

```diff
@@ -0,0 +1,6 @@
+import uiEffects
+
```

Comment:
To maintain the coding style within the application, please remove this empty line.